Dalamud/Interface/Internal/Windows/PluginInstallerWindow.cs

```diff
@@ -1290,6 +1289,7 @@ private void DrawPluginControlButton(LocalPlugin plugin)
                             if (!plugin.IsDev)
                             {
                                 pluginManager.RemovePlugin(plugin);
+                                this.OnAvailablePluginsChanged();
```

Comment:
If we move the filtering of what's installed into this method, freshly installed plugins won't remain in the available tab immediately after installation.

However, if we re-filter after uninstalling, the pluginmaster doesnt re-fetch anything, we only re-filter once, and it seems to just work. This I DID test.